benchmarks/benchmark.py

```diff
@@ -472,7 +474,12 @@ def modify_config_props_for_mms(pargs):
 
     modify_config_props_for_mms(pargs)
 
-    if pargs.all:
+    if pargs.model:
```

Comment:
pargs.model has a default so this makes it impossible to reach the else case (run single benchmark on single model).  Add a new argument to the "target" mutually exclusive group for this additional case

Reply:
Done.